depends/packages/qt.mk

```diff
@@ -69,9 +70,17 @@ $(package)_config_opts += -system-zlib
 $(package)_config_opts += -static
 $(package)_config_opts += -silent
 $(package)_config_opts += -v
+$(package)_config_opts += -no-feature-dial
+$(package)_config_opts += -no-feature-ftp
+$(package)_config_opts += -no-feature-lcdnumber
+$(package)_config_opts += -no-feature-pdf
 $(package)_config_opts += -no-feature-printer
 $(package)_config_opts += -no-feature-printdialog
 $(package)_config_opts += -no-feature-concurrent
+$(package)_config_opts += -no-feature-sql
```

Comment:
`./configure --list-features` output does not list `sql` feature.

Reply:
Passing `-no-feature-sql` disables the Qt sql module in its entirety, see the output in the configure summary:
```
Qt modules and options:
  Qt Network ............................. yes
  Qt Sql ................................. no
  Qt Testlib ............................. yes
```